CHANGELOG.next.asciidoc

```diff
@@ -162,9 +162,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add cost warnings for the azure module. {pull}15356[15356]
 - Release elb module as GA. {pull}15485[15485]
 - Add a `system/network_summary` metricset {pull}15196[15196]
+- Add IBM MQ light-weight Metricbeat module {pull}15301[15301]
 
 *Packetbeat*
 
+- Update DNS protocol plugin to produce events with ECS fields for DNS. {issue}13320[13320] {pull}13354[13354]
```

Comment:
I think this should not be here?

Reply:
Most likely rebase went bad. Resolved now.